docs/release_notes/2024/nov2024.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+id: November_2024
+description: Release notes for November
+title: November 2024
+tags:
+  - release notes
+  - changelog
+  - november
+---
+
+## 3.4.1.\* (November 25, 2024)
+
+- Prophecy Python libs version: 1.9.24
+- Prophecy Scala libs version: 8.4.0
+
+### Features {#Features341}
+
+#### Monitor and alert for resource utilization
+
+Prophecy now offers a reliable solution for configuring alerts to monitor resource usage in Prophecy-managed microservices.
+
+![Alert Config](./img/nov-alert-config.png)
+
+It enables proactive alerting when resource utilization approaches defined thresholds, ensuring timely intervention before limits are reached. Additionally, it supports the suspension of critical services in the event of resource overflows.
+
+For more information, see Alerts Configuration.
```

Comment:
should there be a link here? where do i find "alerts configuration" ?

Reply:
Yes, I'm waiting on the alerts config doc to be merged first. This probably makes a good reason to use release branching in the future.

Reply:
Makes sense. yes, release branches would help with this!